timesketch/api/v1/resources/explore.py

```diff
@@ -197,12 +197,10 @@ def post(self, sketch_id):
 
         # Update or create user state view. This is used in the UI to let
         # the user get back to the last state in the explore view.
-        view = View.get_or_create(
-            user=current_user, sketch=sketch, name='')
+        view = utils.update_sketch_last_activity(sketch, get_view_back=True)
```

Comment:
I think it is better to remove this, and add back `view = View.get_or_create(...)`

**Reason:**
When we change the implementation of `update_sketch_last_activity` and move away from using the `View` for this this will break. Also seeing that this is the only place this is used.

So, move back to:
`view = View.get_or_create()` and `view.update_modification_time()`

Reply:
ack